components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -1457,5 +1483,73 @@ Real cxx_exp(Real input)
 #endif
 }
 
+void cloud_water_conservation_f(Real qc_, Real qcnuc_, Real dt, Real* qcaut_, Real* qcacc_, Real* qccol_,
+  Real* qcheti_, Real* qcshd_, Real* qiberg_, Real* qisub_, Real* qidep_)
+  {
+    using P3F = Functions<Real, HostDevice>;
+
+    typename P3F::Spack qc(qc_);
```

Comment:
You could clean this up by declaring multiple spacks all at once, allowing you to avoid having to repeat the cumbersome type `typename P3F::Spack` over and over.